composer.json

```diff
@@ -12,9 +12,7 @@
     }
   ],
   "require": {
-    "php": ">=5.6"
-  },
-  "require": {
+    "php": ">=7.0",
     "phpunit/phpunit": "5.*"
```

Comment:
@gabriel403 @WillJW Actually, I think we should sort this here. Might as well do it now.